core/src/apps/management/recovery_device/homescreen.py

```diff
@@ -31,6 +31,7 @@ async def recovery_homescreen() -> None:
 
 
 async def recovery_process(ctx: wire.GenericContext) -> Success:
+    wire.AVOID_RESTARTING_FOR = (MessageType.Initialize, MessageType.GetFeatures)
```

Comment:
When recovery finishes, `wire` is unimported and the value is initialized to () next session, right?

Reply:
exactly.